.github/workflows/CentralPublish.yml

```diff
@@ -68,7 +68,7 @@ jobs:
           packagePAT: ${{ secrets.BALLERINA_BOT_TOKEN }}
           GITHUB_TOKEN: ${{ secrets.BALLERINA_BOT_TOKEN }}
         run: |
-          ./gradlew release
+          ./gradlew release -x check -x test
```

Comment:
Is this the correct way? I think it is not okay to skip tests and check, please check other repos.